src/test/java/com/google/devtools/build/lib/skylark/SkylarkDefinedAspectsTest.java

```diff
@@ -904,6 +904,39 @@ public void aspectFailingOrphanArtifacts() throws Exception {
             + "test/missing_in_action.txt\n");
   }
 
+  @Test
+  public void aspectSkippingOrphanArtifactsWithLocation() throws Exception {
+    scratch.file("aspects/BUILD", "");
```

Comment:
nit: you can just put the `.bzl` file in the directory `simple` thereby sparing a line (not that it matters much)

Reply:
Done.